contributing_to_docs/doc_guidelines.adoc

```diff
@@ -69,21 +69,26 @@ There are some general guidelines and conventions we follow for titles, headings
 Links can be used to cross-reference internal topics, or send customers to external information resources for further reading. 
 
 === Internal Cross-References
-Whenever possible the link to another topic should be part of the actual sentence. Avoid creating links as a separate sentence that begins with "See [this topic] for more information on x".
+Whenever possible the link to another topic should be part of the actual sentence. Avoid creating links as a separate sentence that begins with "See [this topic] for more information on x". 
+
+[NOTE]
+====
+Use the full file path, even if the file you're linking to is in the same directory as the file you're writing in; this makes search and replace operations to fix broken links much easier.
```

Comment:
I haven't been able to find a way to use absolute paths in AsciiDoctor, so the "full path" as I believe is referring to here would still have to be a relative path, right? I.e, something like `../../admin_guide/whatever.html`, the point being to ensure that the path includes all the way back to the first topic directory, from wherever relative location the link may be.

If that's the case, we should probably be at least a little more specific than just "Use the full file path".